packages/p2p/src/components/page-return/page-return.tsx

```diff
@@ -10,8 +10,8 @@ type TPageReturnProps = {
 
 const PageReturn = ({ className = '', onClick, page_title }: TPageReturnProps) => {
     return (
-        <div className={classNames('page-return', className)} data-testid='dt_page_return'>
-            <div onClick={onClick} className='page-return__button'>
+        <div className={classNames('page-return', className)}>
+            <div onClick={onClick} className='page-return__button' data-testid='dt_page_return'>
                 <Icon icon='IcArrowLeftBold' size={16} />
```

Comment:
default size is already 16 😜 

```suggestion
                <Icon icon='IcArrowLeftBold'/>
```